src/fe-write.cc

```diff
@@ -68,7 +68,8 @@ int mainWrite(int argc, const char* argv[])
 	if (argc == 1)
 		showProfiles("write", formats);
 	config.mutable_flux_sink()->mutable_drive();
-	config.mutable_flux_source()->mutable_drive();
+	if (verify)
```

Comment:
Yep, this looks like a reasonable fix. It'd be nice to automatically invoke `--no-verify` if the sink isn't hardware-related but that can wait.

Reply:
is this related to / can it be solved by `has_drive()`?  I couldn't actually find where `has_drive` was implemented, it seemed to somehow come from the protobuf stuff.

Reply:
Hmm. I hadn't thought of that. You should be able to test for a hardware sink with `config.flux_sink().has_drive()`, yes, once `parseFlags` is called.